var/spack/repos/builtin/packages/dihydrogen/package.py

```diff
@@ -150,4 +150,13 @@ def cmake_args(self):
                         ' '.join(self.cuda_flags(cuda_arch))
                     ))
 
+        if '+cuda' in spec or '+legacy' in spec:
+            args.extend(['-DcuDNN_DIR={0}'.format(
+                spec['cudnn'].prefix)])
+
+        if spec.satisfies('^cuda@:10.99'):
+            if '+cuda' in spec or '+legacy' in spec:
+                args.extend(['-DCUB_DIR={0}'.format(
+                    spec['cub'].prefix)])
```

Comment:
`list.append(...)` is easier than `list.extend([...])`

Reply:
I updated all of the spots where this is done.